### PR TITLE
#13   varying working hours time formats lead to parser crash

### PR DESCRIPTION
**Pull request by MilanCsore**
Tasks have been done:

- create a unit test for covering this corner case

- add support for parsing/understanding this format as well (should return the same value as the 00.00-24.00 case)

- unsupported/unexpected formats should return an empty string instead of crashing.

In progress:
create a unit test for verifying the return of empty string for unexpected, random string values.

### CODE REVIEW

File: packages/police-hu/src/police-hu.ts [hunk 1]
```diff
@@ -82,47 +82,23 @@ export function extractCrossingNames(htmlContent: string) {
...
-  return [
-  open.trim().replace('.', ':'),
-  close.trim().replace('.', ':')
-  ];
-
-  console.log(open);
-  console.log(close)
-
+  return[
+    openParsedToFloat.toFixed(2).padStart(5, '0').replace('.', ':'),
```
Comment (snorbi07) [inline packages/police-hu/src/police-hu.ts]: Please replace the magic numbers with constants.

File: .editorconfig [hunk 2]
```diff
@@ -6,6 +6,7 @@ root = true
 # begone useless whitespaces
 trim_trailing_whitespace = true
 
+
```
Comment (snorbi07) [inline .editorconfig]: Please remove the committed empty line.
